bin/Caddyfile

```diff
@@ -37,6 +37,7 @@ lessonly.test:8443,
 	}
 	reverse_proxy /skills* 127.0.0.1:3001
 	reverse_proxy /skills/graphql* 127.0.0.1:3002
+	reverse_proxy /readinessBffService* 127.0.0.1:3031
```

Comment:
Few questions.
* We have to do this because of browser [mixed active content](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content#mixed_active_content) issues and we use current dns and https setup upgrade our bff service to https?
* Do we want to extend this pattern? Should we instead use a new domain to mimic production service?  `readiness-bff.test`?